### ISSUE TITLE
Cryptol REPL should say how to get help when it starts

### ISSUE THREAD

**Issue by brianhuffman** [GaloisInc/cryptol#617]
When you start `ghci`, it tells noobs how to get help:
```
GHCi, version 8.6.5: http://www.haskell.org/ghc/  :? for help
```
But `cryptol` doesn't give any such hints:
```
┏━╸┏━┓╻ ╻┏━┓╺┳╸┏━┓╻
┃  ┣┳┛┗┳┛┣━┛ ┃ ┃ ┃┃
┗━╸╹┗╸ ╹ ╹   ╹ ┗━┛┗━╸
version 2.7.1 (ede8c3f, modified)

Loading module Cryptol
Cryptol>
```
I'm thinking about the times I've accidentally started `vim`, couldn't figure out how to quit the program, and then had to open another terminal to kill the process.

Maybe we should add a hint to the startup message about typing `:?` for help.

**Comment by brianhuffman** [comment]
Time for bike-shedding! Should we just add the help hint to the version line, like this:
```
┏━╸┏━┓╻ ╻┏━┓╺┳╸┏━┓╻
┃  ┣┳┛┗┳┛┣━┛ ┃ ┃ ┃┃
┗━╸╹┗╸ ╹ ╹   ╹ ┗━┛┗━╸
version 2.7.1 (ede8c3f) :? for help

Loading module Cryptol
Cryptol>
```
or put an additional line with a URL and the help hint like this:
```
┏━╸┏━┓╻ ╻┏━┓╺┳╸┏━┓╻
┃  ┣┳┛┗┳┛┣━┛ ┃ ┃ ┃┃
┗━╸╹┗╸ ╹ ╹   ╹ ┗━┛┗━╸
version 2.7.1 (ede8c3f)
https://cryptol.net  :? for help

Loading module Cryptol
Cryptol>
```
or maybe something else entirely?

**Comment by atomb** [comment]
My favorite is your second example above, @brianhuffman.

**Comment by brianhuffman** [comment]
Fixed by #657 (revision dd6d8d24c1da3d8f53bde6593fc3a4042c26619a).
